src/vm/gdbjit.cpp

```diff
@@ -2145,6 +2284,8 @@ bool NotifyGdb::BuildDebugInfo(MemBuf& buf, PTK_TypeInfoMap pTypeMap)
 
     for (int i = 0; i < method.GetCount(); ++i)
     {
+        method[i]->lines = lines;
```

Comment:
Are you sure the lifetime of the `lines` is longer than the lifetime of the `method`?